src/main/scala/is/hail/asm4s/CMHelp.scala

```diff
@@ -9,11 +9,11 @@ import scala.collection.mutable
 import scala.reflect.ClassTag
 
 // Without this, we can't use invokeStatic with CompilationHelp as a type.
-case class CompilationHelp() {}
+case class CMHelp() {}
```

Comment:
Yeah, I should have updated the comment too.

Without this line, I cannot refer to `CMHelp` in a type context, in Scala. For example, I couldn't use `CMHelp` as the first type parameter in this expression:
```
Code.invokeStatic[CMHelp, Array[T], mutable.WrappedArray[T]]("__arrayToWrappedArray", a)
```